lang/cs/Org.Apache.REEF.Bridge/InteropUtil.cpp

```diff
@@ -129,6 +129,21 @@ JNIEnv* RetrieveEnv(JavaVM* jvm) {
   return env;
 }
 
+
+const int defaultRecursionDepthForExceptionFormat = 3;
+
 String^ FormatJavaExceptionMessage(String^ errorMessage, Exception^ exception) {
-  return String::Concat(errorMessage, Environment::NewLine, exception->StackTrace);
+	return FormatJavaExceptionMessage(errorMessage, exception, defaultRecursionDepthForExceptionFormat);
+}
+
+String^ FormatJavaExceptionMessage(String^ errorMessage, Exception^ exception, int recursionDepth) {
```

Comment:
was weary to change original signature, but looks like it didn't impact anything, fixed.